build/configs/imxrt1020-evk/loadable_elf_apps/defconfig

```diff
@@ -395,9 +395,9 @@ CONFIG_ARCH_USE_FLASH=y
 #
 CONFIG_FLASH_PARTITION=y
 CONFIG_FLASH_MINOR=0
-CONFIG_FLASH_PART_SIZE="2048,512,3072,128,128,"
-CONFIG_FLASH_PART_TYPE="kernel,none,smartfs,ftl,config,"
-CONFIG_FLASH_PART_NAME="kernel,app,userfs,ftl,config,"
+CONFIG_FLASH_PART_SIZE="2048,3072,128,128,"
+CONFIG_FLASH_PART_TYPE="kernel,smartfs,ftl,config,"
```

Comment:
You'd better divide partition of smartfs, one is for userspace binary, another one is userfs.
Because of wearleveling, middle of user binary can be relocate anytime....
Binary affect to gc condition, performance also decreased significantly.

Reply:
I understand. But, then we need to mount 2 copies of smartfs. Also, there will be changes in the way we generate and flash the smartfs image during the build. I am not very sure about how to implement it. Since, its a big change and not directly related to removal of user binary, we will take this up in a different PR.

Reply:
Ok nice man, but it is really necessary.
I don't mean multi-root dir,(CONFIG_SMARTFS_MULTI_ROOT_DIRS), Make 2 partition, mount each of partition. 
That is, they use different MTD partition.